run.py

```diff
@@ -6,4 +6,4 @@
     format = os.getenv('BEHAVE_FORMAT', 'progress2')
 
     behave_executable.main(args=f'--format {format} system_tests/features')
-    behave_executable.main(args=f'--format {format} acceptance_tests/features')
+    behave_executable.main(args=f'--format {format} acceptance_tests/features/notification_letter.feature')
```

Comment:
This was probably left in for testing locally and can be removed.

Reply:
Yup :D

Reply:
Useful for reviewing though!